test/fixtures/cookbooks/rack_user_test/recipes/remove.rb

```diff
@@ -1,5 +1,10 @@
 # comments!
 
+user 'rack' do
```

Comment:
This is simulating something else creating a user outside of the scope of this library.